zebra-state/src/service.rs

```diff
@@ -942,8 +942,6 @@ impl Service<ReadRequest> for ReadStateService {
         match req {
             // TODO: implement these new ReadRequests for lightwalletd, as part of these tickets
 
-            // z_get_tree_state (#3156)
-
             // depends on transparent address indexes (#3150)
             // get_address_tx_ids (#3147)
             // get_address_balance (#3157)
```

Comment:
Optional: Would you mind removing all these TODOs?

They all got done.

Reply:
Done.